src/table/pagination-table/components/TableWrapper.tsx

```diff
@@ -133,7 +133,7 @@ const TableWrapper = (props: TableWrapperProps) => {
         data-testid="table-container"
       >
         <StyledTable styling={styling} showRightBorder={showRightBorder} stickyHeader aria-label={tableAriaLabel}>
-          <TableHeadWrapper />
+          {viewService.viewState?.skipHeader !== true ? <TableHeadWrapper /> : undefined}
```

Comment:
Cleaner to early return null from `TableHeadWrapper` IMO

Reply:
Good point! Fixed!